src/en/server-hosting/popular-forks-cdns.md

```diff
@@ -0,0 +1,46 @@
+# List of codebases with prebuilt server packages
+
+This document will serve as a list of forks known to have a prebuilt server software for players to download.
+
+This doc was last updated at 2024-10-01 13:52:04 UTC
+
+```admonish danger
+All forks listed here (Exept Wizard's Den) are **not vetted**. We only just put the links in a convinient place.
+
+Third party server builds are not affiliated with the project, and have the samse risks as running any random executable off the internet. Only run them if you trust them to not take over your computer
```

Comment:
```suggestion
Third party server builds are not affiliated with the project, and have the same risks as running any random executable off the internet. Only run them if you trust them to not take over your computer.
```

I would remove the last sentence, or maybe change it to just "Only run software that you trust." There are other bad things that malicious software can do that isn't "take over your computer."

Reply:
Oops someone else wrote this for me and i did not notice this